crates/iroha_executor/src/default/mod.rs

```diff
@@ -44,6 +44,8 @@ use crate::{
     Execute,
 };
 
+pub mod custom;
```

Comment:
```suggestion
pub mod isi;
```

all the stuff here is custom

Reply:
I didn't change this name and it seemed to correspond to `InstructionBox` enum variant. Is this change okay?

Reply:
I think you should change the name to `isi`. Yes, we have one custom isi, but it's like a container for all possible custom isis